### PR TITLE
refactor: use Fondue serializer instead of internal version

### PR DESCRIPTION
**Pull request by ragi96**
If we go with https://github.com/Frontify/web-app/pull/945 we could remove the serializer here and all the hacky classes

### CODE REVIEW

**Comment by SamuelAlev** [comment]
Please add a changeset

File: .changeset/cyan-ghosts-visit.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/guideline-blocks-settings": patch
+---
+
+feat(RTE): use fondue serializer
```
Comment (SamuelAlev) [inline .changeset/cyan-ghosts-visit.md:5]:
```suggestion
refactor: use Fondue serializer instead of internal version
```

**Review by SamuelAlev — APPROVED** [review]

**Review by SamuelAlev — CHANGES_REQUESTED** [review]
On hold as it actually seems to break blocks

**Comment by ragi96** [comment]
> On hold as it actually seems to break blocks

Not related to the changes in this PR, as it breaks (just linked) within the linkPlugin and buttonPlugin. 

A workaround is to use it within blocks as workspace package then it doesn't break. Also with version 0.28.4 it doesn't break, just linked is an issue.

**Review by fulopdaniel — APPROVED** [review]

**Pull request merged**